apps/bfd-data-npi/src/main/java/gov/cms/bfd/data/npi/lookup/NPIOrgLookup.java

```diff
@@ -48,23 +49,31 @@ public static NPIOrgLookup createNpiOrgLookupForTesting() throws IOException {
    * Factory method for creating a {@link NPIOrgLookup } for production that does not include the
    * fake org name.
    *
+   * @throws IOException if there is an issue reading file
    * @return the {@link NPIOrgLookup }
    */
   public static NPIOrgLookup createNpiOrgLookupForProduction() throws IOException {
     if (npiOrgLookupForProduction == null) {
-      npiOrgLookupForProduction = new NPIOrgLookup(false);
+      InputStream npiDataStream = getFileInputStream(App.NPI_RESOURCE);
```

Comment:
Seems a little weird here in that you are invoking a method that contains 1 line. My rule of thumb is:

Create a function if something is done or would be invoked from multiple places. No need to change anything, but seems like you added work since you now have to add javadoc, etc. all for 1 line.

Reply:
Ignore the above comment; traced down to multiple usage(s). Not enough coffee yet this morning!